libs/ui/src/hooks/smartcard_auth/use_dipped_smartcard_auth.test.ts

```diff
@@ -222,26 +311,50 @@ describe('useDippedSmartcardAuth', () => {
       reason: 'invalid_user_on_card',
     });
 
+    cardApi.removeCard();
+    jest.advanceTimersByTime(CARD_POLLING_INTERVAL);
+
     cardApi.insertCard('invalid card data');
     jest.advanceTimersByTime(CARD_POLLING_INTERVAL);
     await waitForNextUpdate();
     expect(result.current).toMatchObject({
       status: 'logged_out',
       reason: 'invalid_user_on_card',
     });
+
+    expect(logger.log).toHaveBeenCalledTimes(2);
```

Comment:
Is the first log call here unrelated to auth and unimportant to this test?

Reply:
No, it would be the same log line, since this test case tests two different card values that cause the same error. I think I just left out the second log call because it wasn't testing any different code path.

Reply:
Ah yes I see now!